cmd/ocm-backplane/cloud/common.go

```diff
@@ -33,7 +41,7 @@ type namedRoleArn struct {
 	Arn  string `json:"arn"`
 }
 
-func getIsolatedCredentials(clusterID string, ocmToken *string) (aws.Credentials, error) {
+func getIsolatedCredentials(clusterID string, queryConfig *CloudQueryConfig, ocmToken *string) (aws.Credentials, error) {
```

Comment:
```suggestion
func (cfg *CloudQueryConfig) getIsolatedCredentials(clusterID string, ocmToken *string) (aws.Credentials, error) {
```

Depending on previous feedback, `clusterID string` would also become a member of `cfg`, same for `isIsolatedBackplaneAccess()`